### ISSUE TITLE
Troubles in starting it up in chrome

### ISSUE THREAD

**Issue by Rolands-Laucis** [peilingjiang/p5.bezier#1]
p5.bezier.min.js:1 Uncaught (in promise) TypeError: Found non-callable @@iterator
    at Module.a (p5.bezier.min.js:1)

Using Svelte framework, p5.min.js, and the web link download of the p5.bezier.js file because i couldnt use the downloadable file. Everything just like the basic example usage. Because when i just <script> add the local file, then it doesnt find the functions and cant use the import keyword since its not a module and require neither. If i do like in the examples with p5bezier.initBezier() then:  Uncaught (in promise) ReferenceError: p5bezier is not defined. Basically couldnt use local file and the downloadable has some error i know nothing about. 
Svelte here is just the basic boilerplate with some p5 drawing custom scripts imported as local files. So i dont think its doing anything destructive.

**Comment by peilingjiang** [comment]
Hi @Rolands-Laucis, I'm not sure of the reason why you have this error. Can you send me a link to your sketch or post the code here? (Also please make sure you're using the lastest version 0.2.4. I updated the whole library around two weeks ago and it's almost completely different from the older ones.)

I also added the example to codepen and here's the link - https://codepen.io/peilingjiang/pen/ZEOLVPx. Hope this will help!

**Comment by peilingjiang** [comment]
I also notice an error in readme that the link to download the file actually points to the unwrapped version of the library so that you might not be able to use it. I've fixed it and you can see the updated instructions here: https://github.com/peilingjiang/p5.bezier/blob/master/README.md#getting-started

**Comment by Rolands-Laucis** [comment]
Okay, thank you! I got it to work from this example. I'm still not sure what the error was about. The only thing i did differently now was how i generated the points array to pass into p5bezier.newBezier(). But for some reason chrome gave me errors about not finding the p5bezier name. Maybe this was due to the Svelte compiler noticing an issue with calling p5bezier.newBezier() with the wrong parameters and deciding not to compile it into the final js file to be run client side, so the browser didnt know what it was. But Svelte didnt give me any compilation errors or warnings. Idk this was a very weird issue, since i did everything pretty much the same. And the errors were about not having the p5bezier file.

Anyway, this is very helpful for my project. Thank you for making this! :) Maybe i could help with it? Like maybe add the 3rd dimension. And simulate pen pressure along the curve. Maybe add support for Chaikin curves (not sure if this is what p5 spline curves are).
